packages/node_modules/node-red/red.js

```diff
@@ -133,9 +133,14 @@ try {
 if (parsedArgs.verbose) {
     settings.verbose = true;
 }
-if (parsedArgs.safe) {
+if (parsedArgs.safe || process.env.NODE_RED_ENABLE_SAFE_MODE) {
     settings.safeMode = true;
 }
+if (process.env.NODE_RED_ENABLE_PROJECTS) {
```

Comment:
As per above - should check for string "false". If we ever make projects enabled the default we will want to be able to set it to false to disable.